src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,363 @@
+@import "../../patternfly-utilities";
+
+.pf-c-sequence {
+  --pf-c-sequence--BackgroundColor: var(--pf-global--BackgroundColor--dark-100); // this background color is interchangable
+
+  // Sequence Control
+  --pf-c-sequence__control--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--BorderTopHeight: var(--pf-global--BorderWidth--lg);
```

Comment:
I might use a pseudo element for this border, so that you can just remove a border when you don't want to show it without causing the content in the border to shift. The edges where the borders meet looks weird now

<img width="120" alt="screen shot 2018-10-10 at 4 02 05 pm" src="https://user-images.githubusercontent.com/35148959/46765904-f3fe4e80-cca5-11e8-8727-3793ca206a04.png">

Reply:
I added the border to a before pseudo element and that fixes that weird style where the borders meet.